rubix-spi/src/main/java/com/qubole/rubix/spi/RetryingPooledBookkeeperClient.java

```diff
@@ -28,23 +29,32 @@
 import org.apache.thrift.transport.TTransport;
 
 import java.io.Closeable;
-import java.io.IOException;
 import java.util.List;
 import java.util.concurrent.Callable;
 
-public class RetryingBookkeeperClient extends BookKeeperService.Client implements Closeable
+public class RetryingPooledBookkeeperClient
+        extends BookKeeperService.Client implements Closeable
 {
-  private static final Log log = LogFactory.getLog(RetryingBookkeeperClient.class);
+  private static final Log log = LogFactory.getLog(RetryingPooledBookkeeperClient.class);
   private int maxRetries;
   TTransport transport;
+  Poolable<TTransport> transportPoolable;
 
-  public RetryingBookkeeperClient(TTransport transport, int maxRetries)
+  public RetryingPooledBookkeeperClient(TTransport transport, int maxRetries)
   {
```

Comment:
remove this constructor. The only user is LocalBookKeeperClient and it could be updated to use Poolable<TTransport> constructor

Reply:
this constructor is used in many tests and creating object of transportpoolable is a little messy, so added annotation of 'VisibleOnlyForTesting'.